ui/src/components/Search/components/SearchResultsContainer/components/SearchResults/SearchResults.js

```diff
@@ -9,20 +9,25 @@ const SearchResults = ({
   getItemProps,
   highlightedIndex,
 }) => (
-  <ListGroup flush {...getMenuProps()}>
-    {results.map((item, index) => (
-      <ListGroupItem
-        {...getItemProps({
-          key: item.nodeId,
-          item,
-          style: { backgroundColor: highlightedIndex === index ? '#f8f9fa' : 'white' },
-        })}
-      >
-        <TypeBadge nodeType={item.__typename} />
-        {item.title || item.name}
-      </ListGroupItem>
-    ))}
-  </ListGroup>
+  // todo: it'd be better to have the menuProps on the ListGroup (i think we might
```

Comment:
@gustavlrsn do you have ideas here? the old code started erroring when i updated `downshift` and this change "fixed" that but yeah, see comment

Reply:
hmm in another place in the reactstrap documentation i found this
```
// Pass in a Component to override default element
  tag: PropTypes.oneOfType([PropTypes.func, PropTypes.string]),
```
and ListGroup has the `tag` prop as well. Would it maybe work to do something like
```
<Listgroup
  tag={ <ul {...getMenuProps()} /> }
>
```

Reply:
hmm no doesn't seem to work to pass an actual component to `tag` (also tried `tag={ () => <ul /> }` but the list didn't render properly then )

Reply:
Moving this to a new issue instead https://github.com/Edgeryders-Participio/realities/issues/103